project/profiles/capacity4more/modules/c4m/demo/c4m_demo_content/csv/user/user.csv

```diff
@@ -7,3 +7,4 @@ user_5,survivalofthefittest,drupal,charles.darwin@evolution.com,Charles,Darwin,"
 user_6,turing,drupal,alan_turing@turingmachine.com,Alan,Turing,"Alan Mathison Turing, (/ˈtjʊərɪŋ/ tewr-ing; 23 June 1912 – 7 June 1954) was a British mathematician, logician, cryptanalyst, computer scientist and philosopher. He was highly influential in the development of computer science, providing a formalisation of the concepts of 'algorithm' and 'computation' with the Turing machine, which can be considered a model of a general purpose computer. Turing is widely considered as the 'Father of Theoretical Computer Science and Artificial Intelligence.",alanturing.gif,Turing Testing Inc.,government,GB,,,United Kingdom,
 user_7,president,drupal,jean@ec.europa.eu,Jean-Claude,Juncker,"The President is the head of the European Commission. According to the Treaties, he decides on the organisation of the Commission, allocates portfolios to members of the Commission and can make changes at any time.",jean.jpg,European Commission,european_institutions,LU,,,,
 user_8,alfrednobel,drupal,alfred@nobel.se,Alfred,Nobel,"He was the inventor of dynamite. His fortune was used posthumously to institute the Nobel Prizes.",alfrednobel.jpg,Nobelprize,private_sector,SE,t_wind|t_earth,t_fire,Sweden|Worldwide,
+user_9,charlesbabbage,drupal,charlie@turingmachine.com,Charles,Babbage,"A mathematician, philosopher, inventor and mechanical engineer, who is best remembered for originating the concept of a programmable computer.",Charles_Babbage_-_1860.jpg,Turing Testing Inc.,government,GB,,,United Kingdom,
```

Comment:
Change the image file name to _charlesbabbage.jpg_ (just to keep the convention).